src/java.desktop/share/classes/sun/awt/util/IdentityArrayList.java

```diff
@@ -388,11 +396,11 @@ public E remove(int index) {
         return oldValue;
     }
 
-    /**
+     /**
      * Removes the first occurrence of the specified element from this list,
-     * if it is present.  If the list does not contain the element, it is
+     * if it is present.  If this list does not contain the element, it is
```

Comment:
```suggestion
     * if it is present.  If this list does not contain the element, the list is
```
I think it adds clarification to _what_ is unchanged.